### ISSUE TITLE
Can't open several instances of the same host many times

### ISSUE THREAD

**Issue by dataviruset** [nergdron/crussh#27]
If I use the crussh command with the same host twice, eg. crussh host1 host1, it will only open one session.

**Comment by nergdron** [comment]
Yeah, internally it deduplicates and sorts hostnames before opening connections. I'm not really sure what the use case is to open multiple connections to the same box, since it'll just send the same input to all of them. Can you give me a better idea what you're trying to do?

**Comment by dataviruset** [comment]
For example if you have a jumpbox you first connect to, then connect to other servers through it. :)

**Comment by nergdron** [comment]
Aha! That makes sense then! The --ssh option should allow this kind of usage (by allowing you to specify a ssh command that includes a jump host) but I can't seem to make it work at the moment, so I'd consider that a bug in that functionality. I'm a little swamped at work ATM, but I'll definitely see what I can do when I get some time. Thanks for the report!

**Comment by nergdron** [comment]
Should also document this functionality in the README once it works.

**Comment by dataviruset** [comment]
Thanks man! It seems to be a great piece of work except for this detail :)

**Comment by nergdron** [comment]
Thanks! Glad to hear you're using it, I wrote it primarily because I was frustrated with cssh, and I know it still needs some polish. This is definitely a useful suggestion.
